cluster-autoscaler/utils/kubernetes/listers.go

```diff
@@ -218,6 +218,30 @@ func NewScheduledPodLister(kubeClient client.Interface, stopchannel <-chan struc
 	}
 }
 
+// ScheduledAndUnscheduledPodLister lists scheduled and unscheduled pods.
+type ScheduledAndUnscheduledPodLister struct {
```

Comment:
To be precise it should be ScheduledAndUnschedulablePodLister (unschedulable instead of unscheduled :) ). Fix in other places as well.